packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio-group/tests/gux-form-field-radio-group.e2e.ts

```diff
@@ -4,13 +4,7 @@ import {
   a11yCheck
 } from '../../../../../../test/e2eTestUtils';
 
-const axeExclusions = [
-  {
-    issueId: 'target-size',
-    exclusionReason:
-      'COMUI-2949 Fix any of the following: Target has insufficient size (16px by 16px, should be at least 24px by 24px); Target has insufficient space to its closest neighbors. Safe clickable space has a diameter of 20px instead of at least 24px.'
-  }
-];
+const axeExclusions = [];
```

Comment:
The new padding values added space between the radio button targets, which resolved the `target-size` accessibility issue for this component. I will update the linked `COMUI-2949` ticket when this is merged.